core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1755,6 +1765,78 @@ class ReplicaManager(val config: KafkaConfig,
     partitionsToMakeFollower
   }
 
+  private def updateTopicIdForFollowers(controllerId: Int,
+                                        controllerEpoch: Int,
+                                        partitionStates: Map[Partition, LeaderAndIsrPartitionState],
+                                        correlationId: Int,
+                                        responseMap: mutable.Map[TopicPartition, Errors],
+                                        topicIds: String => Option[Uuid]) : Set[Partition] = {
+    val traceLoggingEnabled = stateChangeLogger.isTraceEnabled
+    // Do we need this?
```

Comment:
I don't think so. Those logs, which are exactly the same that we have in `makeFollowers`, would be really confusing.